test/components/simple-virtual.riot

```diff
@@ -0,0 +1,5 @@
+<simple-virtual>
+  <p>
+    <virtual>{props.message}</virtual>
```

Comment:
This code is out of the scope for this pull request. Please make another PR to add the `virtual` tag that should be named `template`